t/APISIX.pm

```diff
@@ -384,6 +384,24 @@ _EOC_
         }
     }
 
+    server {
+        listen 1985 ssl;
```

Comment:
We don't need this. We already have a HTTPS backend: 
https://github.com/apache/apisix/blob/3db8ebec269b22394c21e29d1507353bdc9ed52b/t/APISIX.pm#L444
You just need to pass a wrong host header.

Reply:
Thanks. 
But you said I could write a new one in APISIX.pm. What does that means?
![image](https://user-images.githubusercontent.com/18637307/105802506-49c1a680-5fd6-11eb-9bfd-69565a3dbf54.png)